aikau/src/main/resources/alfresco/documentlibrary/_AlfDndDocumentUploadMixin.js

```diff
@@ -625,26 +638,97 @@ define(["dojo/_base/declare",
                   username: null
                };
                var updatedConfig = lang.mixin(defaultConfig, config);
-
-               // Check to see whether or not the generated upload configuration indicates
-               // that an existing node will be created or not. If node is being updated then
-               // we need to generate an intermediary step to capture version and comments...
-               if (updatedConfig.overwrite === false)
```

Comment:
I'm a bit concerned here that it looks like we're losing the ability to perform drag-and-drop upload, is this correct? This would be a regression in behaviour.